packages/components/src/SVGIcon/SVGIcon.stories.tsx

```diff
@@ -54,37 +54,29 @@ const Template: Story<Args> = ({ icon, color, ...rest }) => {
 };
 
 const defaultArgs = {
-  icon: Object.keys(allIcons)[0],
+  icon: Object.keys(allIcons)[0] as keyof typeof allIcons,
```

Comment:
for some reason, this was actually the source of the TS errors 🤔 it was unhappy that icon was type `string`